data/json/monsters/insect_spider.json

```diff
@@ -1356,5 +1356,34 @@
     "death_function": [ "NORMAL" ],
     "special_attacks": [ { "type": "leap", "cooldown": 4, "max_range": 4, "allow_no_target": true }, [ "EAT_CROP", 120 ] ],
     "flags": [ "SEES", "HEARS", "SMELLS", "CLIMBS", "LARVA", "STUMBLES", "PATH_AVOID_FIRE" ]
+  },
+  {
+    "id": "mon_mantis",
+    "type": "MONSTER",
+    "name": { "str": "giant preying mantis" },
+    "description": "A preying mantis, the size of a horse. If you weren't looking closely, you might mistake it for a dead tree.",
+    "default_faction": "mantis",
+    "bodytype": "flying insect",
+    "species": [ "INSECT" ],
+    "volume": "550 L",
+    "weight": "275 kg",
+    "hp": 85,
+    "speed": 160,
+    "material": [ "flesh" ],
+    "symbol": "♠",
+    "color": "green",
+    "aggression": 50,
+    "morale": 30,
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 6,
+    "dodge": 4,
+    "melee_cut": 6,
+    "vision_day": 10,
+    "harvest": "arachnid",
+    "anger_triggers": [ "PLAYER_CLOSE", "STALK" ],
+    "death_function": [ "NORMAL" ],
+    "special_attacks": [ { "type": "leap", "cooldown": 4, "max_range": 4, "allow_no_target": true }, [ "RANGED_PULL", 20 ], [ "GRAB_DRAG", 10 ], [ "SMASH", 30 ]],
```

Comment:
Should probably have a `"difficulty" value of 5-10 to account for the special attacks.